### ISSUE TITLE
Data normalization

### ISSUE THREAD

**Issue by RyanTmi** [RyanTmi/dbdp#1]
## Standardization or Normalization ?
Should we standardize or normalize between what ? [-1,1], [0,1], ...

## What we normalize ?
We normalize at each time step `i`, the dataset $(X_{t_i}^n)_{1\leq n \leq N}$ ?

**Comment by RyanTmi** [comment]
I've tried using `nn.BatchNorm1d` between layers in the `FNN` class, but it doesn't seem to work and takes twice as long to train.
